src/mercury_engine_data_structures/formats/brfld.py

```diff
@@ -11,66 +13,165 @@
 logger = logging.getLogger(__name__)
 
 
+class ActorLayer(Enum):
```

Comment:
this can be `ActorLayer(str, Enum)` so that you can just do `pScenario[actor_layer]` instead of `pScenario[actor_layer.value]` throughout the file